volkr_munko/src/Css/ProductsStyled.js

```diff
@@ -0,0 +1,334 @@
+  background: #fbfbfb;
+  padding: 80px 0 20px;
+  margin: 0 0 100px 0;
+  display: flex;
+  position: relative;
+
+  &::before {
+    content: '';
+    background: url(${productBg}) no-repeat;
+    height: 100px;
+    width: 100%;
+    background-size: 100%;
+    background-position: center top;
+    position: absolute;
+    top: -78px;
+    z-index: 1;
+  }
+
+  @media (max-width: 991px) {
+    padding: 30px 0 40px;
+  }
+`;
+
+export const Container = styled.div`
+  width: 100%;
+  max-width: 1088px;
+  margin: 0 auto;
+
+  &::after {
+    content: '';
+    background: url(${productFooterBg}) no-repeat;
+    height: 158px;
+    width: 100%;
+    background-position: center bottom;
+    background-size: 100% auto;
+    position: absolute;
+    bottom: -150px;
+    left: 0;
+    z-index: 1;
+  }
+
+  @media (max-width: 1199px) {
+    padding: 0 20px;
+  }
+`;
+
+export const ProductsInner = styled.div``;
+
+export const ProductHeading = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+export const ProductMainHeading = styled.div`
+  font-family: 'Quicksand-Bold';
+  font-size: 52px;
+  font-weight: bold;
+  line-height: 76px;
+  letter-spacing: -0.17px;
+  color: #14183e;
+  margin: 0 0 30px 0;
+
+  @media (max-width: 768px) {
+    font-size: 22px;
+  }
+`;
+
+export const ProductDescription = styled.div`
+  font-family: 'Nunito-Regular';
+  font-size: 16px;
+  line-height: 30px;
+  letter-spacing: -0.17px;
+  color: #191919;
+  text-align: center;
+  margin: 0 0 40px 0;
+
+  @media (max-width: 768px) {
+    font-size: 12px;
+  }
+`;
+
+export const ProductButtons = styled.div`
+  display: flex;
+  max-width: 640px;
+  width: 100%;
+  height: 72px;
+  margin: 48px 2px 88px 124px;
+  padding: 0 66px 0 0;
+  border-radius: 36px;
+  box-shadow: 0 3px 6px 0 #00000008;
+  border: solid 1px #eee0e4;
+  background-color: #fff;
+
+  @media (max-width: 768px) {
+    display: none;
+  }
+`;
+
+export const PrivateCustomerButton = styled.div`
+  max-width: 262px;
+  width: 100%;
+  height: 72px;
+  margin: 0 60px 0 0;
+  padding: 0 40px 0 72px;
+  border-radius: 36px;
+  box-shadow: 0 3px 6px 0 #00000008;
+  z-index: 1;
+  background-color: #424244;
+
+  @media (max-width: 768px) {
+    max-width: 40px;
+    width: 100%;
+    height: 32px;
+  }
+`;
+
+export const PrivateCustomerButtonText = styled.div`
+  max-width: 500px;
+  width: 100%;
+  height: 26px;
+  margin: -4px 0 0 13px;
+  font-family: 'Quicksand-Bold';
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 30px;
+  letter-spacing: -0.21px;
+  text-align: center;
+  color: #fff;
+  z-index: 1;
+
+  @media (max-width: 768px) {
+    font-size: 12px;
+    margin: -24px 0 0 -64px;
+  }
+`;
+
+export const PrivateCustomerButtonIcon = styled.div`
+  max-width: 30px;
+  width: 100%;
+  height: 30px;
+
+  img {
+    margin: 20px 0 0 0;
+  }
+
+  @media (max-width: 768px) {
+    img {
+      margin: -30px 0 -10px -36px;
```

Comment:
dont give margin here please remove this and use different property for solve this @aeshapatel025